lib/releases.js

```diff
@@ -79,12 +80,20 @@ const contributorsSentence = ({ commits, pullRequests }) => {
 const getCategoriesConfig = ({ config }) => {
   const categoriesConfig = config.categories.reduce((acc, category) => {
     acc[category.label] = category
+    if (category.labels && category.labels.length) {
+      category.labels.map(label => (acc[label] = category))
+    }
     return acc
   }, {})
 
   const orderedCategories = [
     UNCATEGORIZED,
-    ...config.categories.map(category => category.label)
+    ..._.flatMap(config.categories, category => {
```

Comment:
we could switch to Node JS 12 and gain [`Array.prototype.flat`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/flat)